bot.js

```diff
@@ -10,128 +10,148 @@ var token = config.token;
 
 bot.on('ready', () => {
   console.log('Everything connected!');
-  if(readyspam == 0){
+  if (readyspam == 0) {
     readyspam = 1;
-    setTimeout(function(){
+    setTimeout(function() {
       readyspam = 0;
     }, 3000);
-  }else{
+  } else {
     console.error("Stopping due to client-ready spam, please restart the bot!");
     process.exit(1);
   }
 });
 
 bot.on('message', msg => {
   if (msg.author.id !== bot.user.id) return;
+  if (S(message.content).startsWith('..')) {
+    var cmd = message.content.split(" ")[0];
+    cmd = S(cmd).chompLeft('..').s;
+    var args = message.content.split(" ").slice(1);
 
```

Comment:
Seems fine but i dont know what all of these does, i guess like explode, then substring?
so i guess it's like 
```php
$msg;
$cmd = explode($msg," ");
$cmd = substr(/*remove the .. on start*/);
$args = explode($msg," ");
unset($args[first($args)])
```